transform/mattermost-analytics/models/intermediate/product/configs/int_mm_telemetry_prod__config_plugin.sql

```diff
@@ -0,0 +1,21 @@
+{{
+    config({
+        "materialized": "incremental",
+        "unique_key": ['event_id'],
+        "incremental_strategy": "delete+insert",
+        "cluster_by": ['received_at_date'],
+        "snowflake_warehouse": "transform_l"
+    })
+}}
+
+select 
+     {{ dbt_utils.star(ref('stg_mm_telemetry_prod__plugin')) }}
+     , cast(received_at AS date) AS received_at_date
+     , server_id is not null as has_rudderstack_telemetry_data
```

Comment:
nit: Given that this is a rudderstack source, it can be changed to 
```suggestion
     , true as has_rudderstack_telemetry_data
```

Same goes for segment.

Reply:
4/5 this information might get lost because of the `qualify` expression later in the `dim_daily_server_config`. The `qualify` expression will keep the flag only for the latest record. In most cases this will be OK, since servers usually are either in a really old or a new version. 3/5 the only exception is if a server is upgraded from an old version to a newer one and rolled back.

Reply:
I think we can live with the corner-case, at least for now.